play.pokemonshowdown.com/src/panel-teambuilder-team.tsx

```diff
@@ -107,6 +107,11 @@ class TeamPanel extends PSRoomPanel<TeamRoom> {
 
 	static formatResources = {} as Record<string, FormatResource>;
 
+	static handleDrop = (e: Event) => {
+		PS.teams.save();
+		return true;
+	};
```

Comment:
This should only happen if the thing being dragged is a team. The `return true` part stops the error message, which is bad if you drag an Excel spreadsheet into the teambuilder or something.

Reply:
Also: This should go in `panel-teambuilder`, not `panel-teambuilder-team`.